### PR TITLE
Update App.js

### PR DESCRIPTION
**Pull request by anotherFeng2**

### CODE REVIEW

**Comment by anotherFeng** [comment]
@lab49gittesters

File: src/App.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ class App extends Component {
         <header className="App-header">
           <img src={logo} className="App-logo" alt="logo" />
           <p>
-            Hello World!
+            Hello World
```
Comment (anotherFeng) [inline src/App.js:12]: This is a much better idea!

**Review by anotherFeng — CHANGES_REQUESTED** [review]
Change p tag to div.

File: src/App.js (hunk 1 above)
Comment (anotherFeng2) [reply src/App.js:12]: No can't do.

**Pull request closed without merging**